gulpfile.js

```diff
@@ -48,85 +52,98 @@ function formatByteMessage (source, data) {
   return chalk`{cyan ${source.padStart(12, ' ')}}: {bold ${data.fileName}} ${message}`
 }
 
-function style () {
-  const isLegacy = path => /legacy/.test(path)
+const style = () => {
+  // Don't inline minified versions, so builds can lazily import them at runtime
+  const cssImportOptions = { filter: (path) => !/\.min/.test(path) }
```

Comment:
This is a relict back from when we had an "auto" version that didn't need CSS variables (by importing the entire standalone water.css stylesheet for a given theme depending on the media query).

Can be removed.

Reply:
Boom, removed, thanks.